src/pages/home/report/ReportActionCompose.js

```diff
@@ -417,6 +418,12 @@ class ReportActionCompose extends React.Component {
         if (newComment) {
             this.debouncedBroadcastUserIsTyping();
         }
+
+        const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
+        const exceededCommentLength = hasExceededMaxCommentLength ? this.comment.length : undefined;
+        if (this.state.exceededCommentLength !== exceededCommentLength) {
```

Comment:
Do we still need these checks?

Reply:
sorry, temporarily reverted a commit, and now re-reverted it. All good again, checks removed 😄